reconcile/openshift_tekton_resources.py

```diff
@@ -14,6 +14,7 @@
 from reconcile.utils.openshift_resource import OpenshiftResource as OR
 from reconcile.utils.semver_helper import make_semver
 from reconcile.utils.saasherder import Providers
+from reconcile.utils.sharding import is_in_shard_round_robin
```

Comment:
why did you choose round robin?

Reply:
You got me thinking about an issue that I was not aware of. The list must be sorted consistently. I did not think about that when opening this PR. Let me reconsider using `is_in_shard` or have the list sorted.

Reply:
So assuming qontract-server always returns things in the same order (which is a lot to assume) the potential danger here is the moment which we add a new file and we could have two shards handling the same file, right?

Reply:
I agree with @maorfr that we should use `is_in_shard` FWIW

Reply:
I updated the PR!